src/Runner/DefaultTestResultCache.php

```diff
@@ -183,19 +185,26 @@ public function serialize(): string
         ]);
     }
 
+    /**
+     * @param string $serialized
+     */
     public function unserialize($serialized): void
     {
         $data = \unserialize($serialized);
 
         if (isset($data['times'])) {
+            /** @var float $testTime */
```

Comment:
You're using both `@var` and `assert` for asserting basically the same thing. Nothing wrong with either IMO, but you may want to pick one and use it consistently.

Reply:
Done.